app/uk/gov/hmrc/exports/models/declaration/ExportsDeclaration.scala

```diff
@@ -44,19 +44,26 @@ case class ExportsDeclaration(
 )
 
 object ExportsDeclaration {
-  implicit val formatInstant: OFormat[Instant] = new OFormat[Instant] {
-    override def writes(datetime: Instant): JsObject =
-      Json.obj("$date" -> datetime.toEpochMilli)
-
-    override def reads(json: JsValue): JsResult[Instant] =
-      json match {
-        case JsObject(map) if map.contains("$date") =>
-          map("$date") match {
-            case JsNumber(v) => JsSuccess(Instant.ofEpochMilli(v.toLong))
-            case _           => JsError("Unexpected Date Format. Expected a Number (Epoch Milliseconds)")
-          }
-        case _ => JsError("Unexpected Date Format. Expected an object containing a $date field.")
-      }
+  object REST {
+    implicit val format: OFormat[ExportsDeclaration] = Json.format[ExportsDeclaration]
   }
-  implicit val format: OFormat[ExportsDeclaration] = Json.format[ExportsDeclaration]
+
+  object Mongo {
```

Comment:
Why do we need to create a `Mongo` and `REST` object?
Is it just to separate the `write` and `read` concerns?

As far as naming goes:
One refers to an architecture style `REST`, the other to a NoSQL tool.
Probably it would make sense to think them as layers: `Presentation` Layer vs a `DataAccess` Layer, but naming is always difficult.

Reply:
The JSON that it serializes to is subtly different, e.g. Mongo stored dates differently.

I've named the object `Mongo` as the format is very specific to that type of DB, If we swapped it out for a different NoSQL DB I'd imagine the format would be completely differentt